app/utils/url.js

```diff
@@ -43,6 +43,26 @@ export function isYoutubeLink(link) {
     return link.trim().match(ytRegex);
 }
 
+export async function getShortenedImageLink(link) {
+    return new Promise(((resolve, reject) => {
+        const xhr = new XMLHttpRequest();
```

Comment:
It might be a good idea to add an `Accept: image/*` header onto the request since right now it just requests whatever payload

Reply:
@hmhealey Also a good idea.